src/components/User/User.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
```

Comment:
`User` seems like a pretty generic name. Maybe `UserMenu`? This also might be a problem with the `user` class below being pretty generic which would be solved by going to `user-menu`.